logappender/README.md

```diff
@@ -0,0 +1,64 @@
+# RocketMQ-LogAppender   [![Build Status](https://travis-ci.org/rocketmq/rocketmq-logappender.svg?branch=master)](https://travis-ci.org/rocketmq/rocketmq-logappender) [![Coverage Status](https://coveralls.io/repos/github/rocketmq/rocketmq-logappender/badge.svg?branch=master)](https://coveralls.io/github/rocketmq/rocketmq-logappender?branch=master)
+
+
+## Introduction
+RocketMQ-LogAppender is a logging component implement of log4j,log4j2 and logback.Taking Apache RocketMQ as broker.
```

Comment:
What is "implement of " It's confused. IMO, this project is used to send log message using RocektMQ. while log4j, log4j2 and logback is just a  adapter for SL4J